orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java

```diff
@@ -321,6 +329,28 @@ public void validateRegistrationFields(HttpServletRequest request, Registration
         regEmailValidate(request, reg, false, false);
         registerTermsOfUseValidate(reg);
 
+        if (Features.EVENTS.isActive() && reg.getAffiliationForm() != null) {
```

Comment:
@DanielPalafox again, I think the feature `Features.EVENTS` is not the one we want to use here, could you please create one, something like `Features.AFFILIATIONS_ON_REGISTRATION`

Reply:
Done